src/app/modules/backbone/components/samples-confirmation-modal/samples-confirmation-modal.component.html

```diff
@@ -7,7 +7,12 @@
         </div>
         <div class="modal-body-content">
             <h3 class="modal-body-content-header" id="modal-title">Order samples</h3>
-            <p class="modal-body-content-message">Are you sure you want to order samples?</p>
+            <p class="modal-body-content-message">Select blend:</p>
+            <div class="modal-body-content-selection-item" *ngFor="let option of selectionOptions">
+                <input id="{{ option }}" name="sampleSelection" type="radio" value="{{ option }}" (click)="onSelection(option)" class="modal-body-content-selection-input" />
+                <label for="{{ option }}" (click)="onSelection(option)" class="modal-body-content-selection-label">{{ option }}</label>
+            </div>
+            <div class="modal-body-no-selection" [ngClass]="{ 'modal-body-no-selection-show': submissionAttempted && selection == undefined }">Please select a blend before submitting, or click "Cancel".</div>
```

Comment:
check the login component to see how to display errors using the html (and also how to use reactive forms again)

example: https://www.concretepage.com/angular-2/angular-2-radio-button-and-checkbox-example#reactive-form